### ISSUE TITLE
Can't write logs from Angular level

### ISSUE THREAD

**Issue by vasakorol** [megahertz/electron-log#307]
Hello,

i work with electron (v16) and  angular (v13.0.1).

I'm using electron-log to log from electron level and from angular level apps.

For electron level eveyrthing work fine,

But for angular level app there is strange problem, file were created, but nothing is written there, and i get such warning into dev console:

```
19:44:52.641 › electron-log.transports.file: Can't write to [C:\Users\vasyl\AppData\Roaming\app_folder\logs\1234.log] TypeError: Cannot read properties of null (reading '1')
    at checkNodeJsVersion (http://localhost:4200/main.js:22467:65)
    at mkDir (http://localhost:4200/main.js:22433:7)
    at FileRegistry.testFileWriting (http://localhost:4200/main.js:22426:3)
    at FileRegistry.createFile (http://localhost:4200/main.js:22406:8)
    at FileRegistry.provide (http://localhost:4200/main.js:22386:17)
    at getFile (http://localhost:4200/main.js:22593:21)
    at transport (http://localhost:4200/main.js:22545:16)
    at runTransport (http://localhost:4200/main.js:21401:5)
    at runTransports (http://localhost:4200/main.js:21384:7)
    at log (http://localhost:4200/main.js:21378:3)
```

Even if i try to log into same file, from both levels, it work fine for electron level, but not for angular.

Does anyone had such problem?

Thanks for any feedback

**Comment by megahertz** [comment]
Seems that node.js version detection may work incorrectly after a bundler. Should be fixed in v4.4.7

**Comment by vasakorol** [comment]
@megahertz thanks
